service/http.directory/src/test/webapp/symlink/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
```

Comment:
Im not sure about this file either, it was the original welcomefile with nothing in it, but I am not sure what i changed to it